### PR TITLE
Bash functions (part 2)

### PR DESCRIPTION
**Pull request by OneCricketeer**
Curious what your thoughts are on Makefile or [`pacur`](https://github.com/pacur/pacur)

### CODE REVIEW

File: .github/scripts/linux.sh [hunk 1]
```diff
@@ -50,24 +52,9 @@ jpackage --name "$CDK_APP_NAME" \
               --main-jar "desktop-$VERSION.jar" \
               --runtime-image "$CUSTOM_JRE_NAME" \
               --java-options "$CDK_JAVA_OPTIONS"
-mv conduktor_$VERSION*.deb "Conduktor-$VERSION.deb"
+  mv "conduktor_$VERSION*.$TYPE" "Conduktor-$VERSION.$TYPE"
```
Comment (sderosiaux) [inline .github/scripts/linux.sh:55]: globbing won't work with the quotes

**Comment by sderosiaux** [comment]
Thanks for the PR (left a comment).
About make, i personally do not want to use this. About pacur, we don't know this tool, we'll check this out. We're probably also going to use flatpak (a PR is waiting thanks to another contributor), it seems to cover the same ground, do you know this one?

**Comment by OneCricketeer** [comment]
Oh, completely missed the glob. Will fix 

Flatpak or Snapcraft would be good, yeah. Never made one

I also tried to do pacur once but ended up using fpm due to lack of documentation 

Make is mostly just a shell. If you're packaging Java code, Ant would be another option, or I've seen Gradle plugins that build Linux packages as well

Up to you

**Comment by sderosiaux** [comment]
Thanks for the contribution, closing it for now

**Pull request closed without merging**